boards/norik/octopus_io_board/doc/index.rst

```diff
@@ -0,0 +1,147 @@
+.. _octopus_io_board:
+
+Norik Octopus IO-Board
+######################
+
+Overview
+********
+
+Octopus IO-Board is an expansion to the Octopus SoM, which is built around the nRF9160 SiP
+offering NB-IoT and LTE-M connectivity, GPS and accelerometer. Octopus IO-Board expands
+the capabilities of the Octopus SoM by providing additional peripherals and interfaces for
+development and prototyping of low-power IoT applications.
+
+nRF9160 SiP contains ARM Cortex-M33 application processor and the
+following devices:
+
+* :abbr:`ADC (Analog to Digital Converter)`
+* CLOCK
+* FLASH
+* :abbr:`GPIO (General Purpose Input Output)`
+* :abbr:`I2C (Inter-Integrated Circuit)`
+* :abbr:`MPU (Memory Protection Unit)`
+* :abbr:`NVIC (Nested Vectored Interrupt Controller)`
+* :abbr:`PWM (Pulse Width Modulation)`
+* :abbr:`RTC (nRF RTC System Clock)`
+* Segger RTT (RTT Console)
+* :abbr:`SPI (Serial Peripheral Interface)`
+* :abbr:`UARTE (Universal asynchronous receiver-transmitter with EasyDMA)`
+* :abbr:`WDT (Watchdog Timer)`
+* :abbr:`IDAU (Implementation Defined Attribution Unit)`
+
+.. figure:: img/Norik_Octopus_IO-board.webp
+      :align: center
+      :alt: Norik Octopus IO-Board front view
+
+      Norik Octopus IO-Board
```

Comment:
drop - zephyr:board directive will take care of showing the image in a sidebar "a la wikipedia".

```suggestion
```

Reply:
Fixed